pkg/webhooks/validation/controller/controller.go

```diff
@@ -366,6 +374,10 @@ func (c *Controller) updateValidatingWebhookConfiguration(caBundle []byte, failu
 	for i := range updated.Webhooks {
 		updated.Webhooks[i].ClientConfig.CABundle = caBundle
 		updated.Webhooks[i].FailurePolicy = &failurePolicy
+		// Update the mesh config webhook to only validate Istio mesh config of the specified revision.
```

Comment:
I don't think this will work well, unless I misunderstand it? There is one Istiod that will be the leader. That means if we have 3 revisions, only one of their configmaps will be validated - the others will have no validation at all

Reply:
Oh I see you already mentioned that, should have read the PR description :slightly_smiling_face: